moVirt/src/main/java/org/ovirt/mobile/movirt/ui/AuthenticatorActivity.java

```diff
@@ -364,6 +368,14 @@ void onLoginResultReceived(String token, boolean endpointChanged) {
         finish();
     }
 
+    @Background
+    public void deleteCaFile() {
```

Comment:
duplicate code also in https://github.com/matobet/moVirt/blob/master/moVirt/src/main/java/org/ovirt/mobile/movirt/ui/AdvancedAuthenticatorActivity.java#L377
maybe refactoring?